modules/300-prometheus/monitoring/prometheus-rules/deckhouse.yaml

```diff
@@ -0,0 +1,15 @@
+- name: d8.deckhouse
+  rules:
+  - alert: DeprecatedDeckhouseConfigmapManagedByArgoCD
+    expr: |
+      d8_deprecated_configmap_managed_by_argocd > 0
+    labels:
+      tier: cluster
+      severity_level: "4"
+    annotations:
+      plk_markup_format: markdown
+      plk_protocol_version: "1"
+      for: "10m"
+      summary: Deprecated deckhouse configmap managed by Argo CD
+      description: |
+        The deckhouse configmap is no longer used.
```

Comment:
I think we need to add more information about fix this (remove from argo or another things)

Reply:
Done